CollAction/Frontend/app/global/ShareButtons.tsx

```diff
@@ -14,98 +14,138 @@ interface IShareButtonsState {
   shareCount?: number;
   shareCountError?: Error;
   shareCountState: States;
+    width: number;
```

Comment:
The indentation of the whole file and changeset seems a bit inconsistent. Make sure you try to emulate the indentation style of the other typescript files.